README.md

```diff
@@ -160,8 +160,8 @@ Please consult [the documentation](https://github.com/operator-framework/operato
 | vendor | The golang [Vendor](https://golang.org/cmd/go/#hdr-Vendor_Directories) folder that contains the local copies of the external dependencies that satisfy the imports of this project. [dep](https://github.com/golang/dep) manages the vendor directly.|
 
 
-## Enabling the DevOps perspective in OpenShift
+## Enabling the Dev perspective in OpenShift
```

Comment:
It would be better If use "Developer" here.

Reply:
Good point, @baijum .